WISE-Community/WISE
# Added Github icons to homepage and footer + legacy site link to footer

To test:
- Load homepage.
- Make sure links to Github and legacy site on homepage and footer work.


Closes #1755.
